README.md

```diff
@@ -54,6 +54,32 @@ make_it_so sinatra <app_name>
 
 By default, the generator, will create a sinatra root complete with an RSpec configuration.
 
+### Gosu
+
+In the terminal, run:
+```no-highlight
+make_it_so gosu <app_name>
+```
+
+By default, the generator, will create a gosu template complete with an RSpec configuration.
+
+The tree structure looks like:  
+
+```no-highlight  
+GosuGame
+├── Gemfile
+├── README.md
+├── game.rb
+├── img
+├── lib
+│   ├── bounding_box.rb
```

Comment:
I'd probably omit these files to reflect the filesystem as it sits post invocation